### PR TITLE
Flips container on landing page.

### PR DESCRIPTION
**Pull request by Parikshit85**
# Pull Request
related to #5
## Description
- Add a section on landing page of flow that shows flips of that particular block chain community.
-  Change BrowseCommunityButton Component -> generalised navigation  button component.

## Affected Modules
- Main Landing page will now have a section called the FLIPS (Flow Improvement Proposal) which will show the latest 3 FLIPS.

## Things to consider - 
- All the proposals are FLIPS inside the blockchain community.
- Community ID for the main community is a constant with value 1.
- Always there will be an image inside the description of a particular FLIP. If not, we are providing a default image. (@vishalchangrani TBD)
- The FLIPS are shown in the order of Active flips first and terminated flips in the end if any condition met.
- FLIP tile appears at the top level but below the banner (If there's any).
- If > 3 FLIPs are live show the top-3 arranged by end-date (end-date is the closest to current date).
- If < 3 FLIPs are live, show the live ones + most recently finished/ concluded FLIPs. 

## Checklist
- [x] I have tested the changes locally and they are functioning as expected.
- [x] I have assigned the PR to the appropriate reviewer(s) for their feedback.

### CODE REVIEW

File: frontend/packages/client/src/components/FlipsContainer/FlipCardFooter.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  [FilterValues.pending]: (
+    <button
+      class={`button is-rounded is-small has-text-weight-semibold has-background-orange has-border-orange`}
+    >
+      Upcoming...
+    </button>
+  ),
+  [FilterValues.closed]: (
+    <button
+      class={`button is-rounded is-small has-text-weight-semibold has-background-green has-border-green`}
```
Comment (AmarildoGrembi) [inline frontend/packages/client/src/components/FlipsContainer/FlipCardFooter.js]: Please lets use the same styling as in the community page for the buttons for consistency, for completed Flips its `has-background-success`

**Review by AmarildoGrembi — CHANGES_REQUESTED** [review]

File: frontend/packages/client/src/components/FlipsContainer/FlipCardFooter.js (hunk 1 above)
Comment (Parikshit85) [reply frontend/packages/client/src/components/FlipsContainer/FlipCardFooter.js]: The color of complete button I've made consistent with the website green theme.
In community page, a new dark green color is breaking the flow. Attaching images for reference.
Please let me know if still we should change that to dark green.
Screenshot 2023-08-18 at 12 08 59 PM

Comment (Parikshit85) [reply frontend/packages/client/src/components/FlipsContainer/FlipCardFooter.js]: Need to add the tick icon after complete

Comment (Parikshit85) [reply frontend/packages/client/src/components/FlipsContainer/FlipCardFooter.js]: @AmarildoGrembi I've added button for active status and badges for other statuses.
Please check and let me know if we can improve anything.

Comment (AmarildoGrembi) [reply frontend/packages/client/src/components/FlipsContainer/FlipCardFooter.js]: Thats a great idea actually, thanks! One last thing, once a user casts a vote - could we please show the badge + days remaining for the proposal to be completed, similar to how its done in the community page?

**Review by AmarildoGrembi — APPROVED** [review]
LGTM, thanks!

**Review by vishalchangrani — APPROVED** [review]

**Pull request merged**